holobot/discord/servers/managers/user_manager.py

```diff
@@ -1,13 +1,45 @@
+from datetime import datetime, timedelta, timezone
+
 from hikari import ForbiddenError as HikariForbiddenError
 
 from holobot.discord.sdk.exceptions import ForbiddenError
 from holobot.discord.sdk.servers.managers import IUserManager
 from holobot.discord.utils import get_guild_member, get_guild_role
+from holobot.sdk.exceptions import ArgumentError
 from holobot.sdk.ioc.decorators import injectable
-from holobot.sdk.utils import assert_not_none, assert_range
+from holobot.sdk.utils import assert_not_none, assert_range, textify_timedelta
+
+_DEFAULT_TIME_OUT_DURATION = timedelta(minutes=1)
+_MIN_TIME_OUT = timedelta(minutes=1)
```

Comment:
I don't see the reason this has to be 1 minute. Discord allows even 1 second timeouts.

Reply:
I'd like to avoid wasting expensive resources on pointless usage. Responding is required anyway, but wasting another request to Discord on muting for 1 second just doesn't seem right. What do you think?

Reply:
You are creating restrictions for developers based on a vague assumption of how the feature is going to be used. If someone wants to make an API call, it's their right to do so.

Reply:
You're absolutely right, I'll change it. Nice one. 👍